tests/Avalonia.Markup.Xaml.UnitTests/Xaml/BasicTests.cs

```diff
@@ -124,6 +125,24 @@ public void Attached_Property_In_Panel_Is_Set()
             Assert.Equal("Foo", ToolTip.GetTip(target));
         }
 
+        [Fact]
+        public void NonExistent_Property_Throws()
+        {
+            var xaml =
+        @"<ContentControl xmlns='https://github.com/avaloniaui' DoesntExist='foo'/>";
+
+            Assert.Throws<XamlObjectWriterException>(() => AvaloniaXamlLoader.Parse<ContentControl>(xaml));
+        }
+
+        [Fact]
+        public void Non_Attached_Property_With_Attached_Property_Syntax_Throws()
+        {
+            var xaml =
+        @"<ContentControl xmlns='https://github.com/avaloniaui' TextBlock.Text='foo'/>";
```

Comment:
It would be nice if we could eventually support this for inheriting properties. I believe WPF supports this. It makes it cleaner to set a property in a parent control and have it propagate to all relevant children.

Reply:
That should already work after this PR - if not it's a bug... Do you have an example of it not working?